tools/packaging/kata-deploy/local-build/kata-deploy-build-and-upload-payload.sh

```diff
@@ -15,10 +15,17 @@ cp ${KATA_DEPLOY_ARTIFACT} ${KATA_DEPLOY_DIR}
 
 pushd ${KATA_DEPLOY_DIR}
 
-IMAGE_TAG="${REGISTRY}:kata-containers-$(git rev-parse HEAD)"
+IMAGE_TAG="${REGISTRY}:kata-containers-$(git rev-parse HEAD)-$(uname -m)"
 
 echo "Building the image"
-docker build --tag ${IMAGE_TAG} .
+if [ "$(uname -m)" = "s390x" ]; then
+	docker build \
+		--build-arg IMG_NAME=fedora \
+		--build-arg IMG_TAG=37 \
```

Comment:
This worries me a little bit.

Were you able to actually use the image?  I remember having issues with running systemd inside a container with newer versions of Fedora / RHEL / CentOS, and that's the main reason we decided to stick to the CentOS 7 image.

Maybe sticking to ClefOS 7 would be a better deal, in general, as it'd match better with what we use for the amd64 one.

Reply:
Unfortunately, an image `centos` is not supported for s390x atm. I changed it because I got an error for this during the build.

Reply:
I will give it a try with clefos then. Thanks for the suggestion. ^^

Reply:
I have updated the image to `clefos:7` after testing on a s390x machine. Thanks a lot!